config/crd/bases/kuadrant.io_dnshealthcheckprobes.yaml

```diff
@@ -45,22 +45,23 @@ spec:
           spec:
             description: DNSHealthCheckProbeSpec defines the desired state of DNSHealthCheckProbe
             properties:
-              AllowInsecureCertificate:
-                type: boolean
               additionalHeaders:
-                items:
-                  properties:
-                    name:
-                      type: string
-                    value:
-                      type: string
-                  required:
-                  - name
-                  - value
-                  type: object
-                type: array
+                properties:
```

Comment:
this is a bit confusing. Am I refrencing a secret?

Reply:
yes, this is following @roivaz suggestion, in case the headers contained an authorization token or similar.

Reply:
ok I see, I think then it should be consistent with other refs
IE for tls it is
```certificateRefs```
so for headers should prob be 
```additionalHeadersRef``` ?